components/colorwheel/index.css

```diff
@@ -10,126 +10,128 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 .spectrum-ColorWheel {
-  --spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
-  --spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
-  --spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
-  --spectrum-colorwheel-border-color: var(--spectrum-transparent-black-200);
-  --spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
-  --spectrum-colorwheel-fill-color-disabled: var(--spectrum-disabled-background-color);
-  --spectrum-colorwheel-track-width: var(--spectrum-color-control-track-width);
-  --spectrum-colorwheel-colorarea-margin: var(--spectrum-color-wheel-color-area-margin);
-  --spectrum-colorwheel-colorhandle-position: calc((var(--spectrum-colorwheel-width) / 2) - (var(--spectrum-colorwheel-track-width) / 2));
+	--spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
+	--spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-border-color-rgb: var(--spectrum-gray-1000-rgb);
+	--spectrum-colorwheel-border-opacity: 0.1;
+	--spectrum-colorwheel-border-color: rgba(var(--spectrum-colorwheel-border-color-rgb), var(--spectrum-colorwheel-border-opacity));
+	--spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
```

Comment:
Hm...so everything checks out right now. The border width is 1px, and if I inspect the Figma component, it also has a 1px border.

<img width="243" alt="Screenshot 2024-11-13 at 4 16 07 PM" src="https://github.com/user-attachments/assets/a2b0b77e-e875-4f10-8d16-8e97156b967a">

However, I read [the blurb in the desktop file](https://www.figma.com/design/Mngz9H7WZLbrCvGQf3GnsY/S2-%2F-Desktop?node-id=20606-73), and it says that the color wheel is supposed to have a 2px border? You may have already done this, but I was going to suggest we double check this with design. This change isn't listed in the [changelog in Figma](https://www.figma.com/design/M9P2s2IDhGWZuNyvJtXrx3/S2-foundations-and-components-changelog?node-id=1186-2577&t=IaB6l2T1tpC91R97-4), so I _think_ maybe this is outdated information. Maybe design would want to correct that 2px thing? Or update `border-width-100`?

Reply:
I went ahead and added a comment about this to the thread I dropped into the implementations channel. I'll get both updated pending some clarification. ✨